pkg/providers/instance/skuclient/skuclient.go

```diff
@@ -48,11 +50,14 @@ func (sc *skuClient) updateInstance() {
 	sc.mu.RLock()
 	defer sc.mu.RUnlock()
 
-	authorizer, err := auth.NewAuthorizer(sc.cfg, sc.env)
+	// Create a new authorizer for the sku client
+	// TODO (charliedmcb): need to get track 2 support for the skewer API
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
 	if err != nil {
-		klog.V(5).Infof("Error creating authorizer for sku client: %s", err)
+		klog.V(5).Infof("Error creating authorizer for sku client: default cred: %s", err)
 		return
 	}
+	authorizer := azidext.NewTokenCredentialAdapter(cred, []string{azidext.DefaultManagementScope})
```

Comment:
An alternative is to keep the abstraction, that returns authorizer. But this is the only use of it at the moment, so I don't think it is worth it. Could refactor later.